### ISSUE TITLE
Consolidating Phenotype 3.1 Changes

### ISSUE THREAD

**Issue by empfff** [National-COVID-Cohort-Collaborative/Phenotype_Data_Acquisition#181]
Use the buddy-saver branch for phenotype 3.1.
--Emily: add buddy saver code to ACT if necessary
--Marshall: add buddy saver to MS SQL version of PCORnet script
--Already done (in branch): weak dx fix to OMOP script (MSSQL version)
             --IF YOU USE SQL RENDER FIX PROBLEMS AFTERWARDS PLLLLSSSS
--ALL: add new LOINC codes to LOINC list from #177 
--ALL: add new ICD-10 codes to ICD-10 list (dx_strong) from #175
--Robert: adjust OMOP extract script to extract month and year of birth from full birthdate field
--Kristin: double-check that there isn't a stray observation_concept_id in the list of condition_concept_ids

**Comment by marshallclark** [comment]
Buddy saver code added to ACT oracle, ACT mssql, PCORnet oracle, PCORnet mssql

**Comment by empfff** [comment]
new LOINC and ICD-10s added to all ACT and PCORnet scripts on buddy saver branch

**Comment by kmkostka** [comment]
OMOP SQL server version has addressed:
- add new LOINC codes to LOINC list from #177 
- created new OMOP-specific issue #182 to address #175 when OMOP vocabulary tables catch up
- removed stray observation_concept_id in the list of condition_concept_ids and added logic to pull observation_concept_id

Before we can SQL Render, need @rtmill to adjust OMOP extract script to extract month and year of birth from full birthdate field.

**Comment by kmkostka** [comment]
Pushed updates to SQL render and verified datediff issue is not occurring in GBQ, Oracle, Postgres and Redshift conversions.

**Comment by empfff** [comment]
Reopening--will close once @LoraLingrey-TriNetX has pushed changes.

**Comment by LoraLingrey-TriNetX** [comment]
@empfff - Updates made in TNX query
